convert2rhel/actions/pre_ponr_changes/handle_packages.py

```diff
@@ -60,16 +80,44 @@ def run(self):
         logger.task("Convert: Remove excluded packages")
         logger.info("Searching for the following excluded packages:\n")
         try:
-            pkgs_to_remove = pkghandler._get_packages_to_remove(system_info.excluded_pkgs)
-            pkghandler.remove_pkgs_unless_from_redhat(pkgs_to_remove)
+            pkgs_to_remove = sorted(pkghandler.get_packages_to_remove(system_info.excluded_pkgs))
+            # this call can return None, which is not ideal to use with sorted.
+            pkgs_removed = sorted(pkghandler.remove_pkgs_unless_from_redhat(pkgs_to_remove) or [])
```

Comment:
Ideally we should change that function so we always return an array. Makes life much easier

Reply:
We can remove the `or []` in a subsequent PR.